src/components/EligibilityForm/components/ContactInformation.tsx

```diff
@@ -119,43 +173,99 @@ const ContactInformation = ({
+        <InputWraper className={fr.cx('fr-fieldset__element')} my="1w">
+          <Text mb="1w" fontSize="14px" fontWeight="bold">
+            {fieldLabelInformation.contactDetailsTitle}
+          </Text>
+        </InputWraper>
         <InputWraper className={fr.cx('fr-fieldset__element')} my="1w">
           <Field name="lastName" required label={fieldLabelInformation.lastName} component={Input} />
         </InputWraper>
         <InputWraper className={fr.cx('fr-fieldset__element')} my="1w">
           <Field name="firstName" required label={fieldLabelInformation.firstName} component={Input} />
         </InputWraper>
-        {values.structure === 'Tertiaire' && (
-          <InputWraper className={fr.cx('fr-fieldset__element')} my="1w">
-            <Field name="company" label={fieldLabelInformation.company} component={Input} required />
-          </InputWraper>
-        )}
         <InputWraper className={fr.cx('fr-fieldset__element')} my="1w">
           <Field name="email" type="email" label={fieldLabelInformation.email} component={Input} required />
         </InputWraper>
         <InputWraper className={fr.cx('fr-fieldset__element')} my="1w">
           <Field name="phone" placeholder="0605040302" label={fieldLabelInformation.phone} component={Input} />
         </InputWraper>
       </fieldset>
+      {values.structure === 'Tertiaire' && (companyType === "Bureau d'études ou AMO" || companyType === 'Mandataire / délégataire CEE') && (
+        <fieldset className={fr.cx('fr-fieldset')}>
+          <InputWraper className={fr.cx('fr-fieldset__element')} my="1w">
+            <Select
+              label={fieldLabelInformation.demandCompanyType.label}
+              options={fieldLabelInformation.demandCompanyType.inputs}
+              nativeSelectProps={{
+                required: true,
+                onChange: (e) => setDemandCompanyTypeValue(e.target.value),
+              }}
+            ></Select>
+          </InputWraper>
+          {demandCompanyType !== 'Copropriété' && demandCompanyType !== 'Maison individuelle' && (
+            <InputWraper className={fr.cx('fr-fieldset__element')} my="1w">
+              <Field name="demandCompanyName" label={fieldLabelInformation.demandCompanyName} component={Input} />
+            </InputWraper>
+          )}
+          {demandCompanyType === 'Bâtiment tertiaire' && (
+            <InputWraper className={fr.cx('fr-fieldset__element')} my="1w">
+              <Field name="demandArea" type="number" label={fieldLabelInformation.demandArea} component={Input} />
+            </InputWraper>
+          )}
+        </fieldset>
+      )}
+      {(values.structure === 'Copropriété' ||
+        (values.structure === 'Tertiaire' && (demandCompanyType === 'Copropriété' || demandCompanyType === 'Bailleur social'))) && (
+        <fieldset className={fr.cx('fr-fieldset')}>
+          <InputWraper className={fr.cx('fr-fieldset__element')} mb="1w">
```

Comment:
```suggestion
          <InputWraper className={fr.cx('fr-fieldset__element')} my="1w">
```

Pour suivre la logique du reste.

Reply:
Si je mets my ça créé un trop grand espace et le bloc se retrouve séparé du reste. D'où le bottom uniquement.

Reply:
sinon au lieu d'utiliser des merci beaucoup, on peut aussi faire un parent flex column avec un gap (en utilisant Box)
Sinon dans `reorg_carte` j'ai ajouté des `fr-gap--sm` car je trouve que ca manque dans le DSFR